server/src/test/java/org/apache/druid/segment/indexing/DataSchemaTest.java

```diff
@@ -54,10 +57,14 @@
 import java.nio.charset.StandardCharsets;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.List;
 import java.util.Map;
 
-public class DataSchemaTest
+public class DataSchemaTest extends NullHandlingTest
```

Comment:
nit: it's actually `InitializedNullHandlingTest` that you want, but the one you've extended also initializes null handling